components/TreeHeatmap/test/treeheatmap.js

```diff
@@ -0,0 +1,46 @@
+import test from 'tape-catch';
+import TreeHeatmap from '..';
+
+test('TreeHeatmap component ', t => {
+  t.equal(TreeHeatmap.options[0].type, 'table');
+
+  let el = document.createElement('div');
+  let vis = new TreeHeatmap(el);
+  vis.render();
+  t.equal(el.childNodes.length, 0, 'empty options should do nothing');
+
+  vis = new TreeHeatmap(el, {data: []});
+  vis.render();
+  t.equal(el.childNodes.length, 0, 'empty data should do nothing');
+
+  vis = new TreeHeatmap(el, {data: [{}]});
+  vis.render();
+  t.equal(el.childNodes.length, 0, 'no idColumn should do nothing');
+
+  vis = new TreeHeatmap(el, {data: [{_id: 0, a: 0}]});
+  vis.render();
+  t.equal(el.childNodes.length, 1, 'single row and column should produce something');
+  console.log(el.innerHTML);
+  let single = document.createElement('div');
+  single.innerHTML = '<svg width="400px" height="400px"><clipPath id="clip-rect"><rect x="100" y="100" width="200" height="200"></rect></clipPath><g clip-path="url(#clip-rect)"><rect class="datum" fill="rgb(255, 255, 255)" x="100" y="100" opacity="1" width="200" height="200"></rect></g><clipPath id="clip-row-labels"><rect x="300" y="100" width="100" height="200"></rect></clipPath><g clip-path="url(#clip-row-labels)"><text class="row-label" color="black" font-size="10px" alignment-baseline="middle" x="300" y="200">0</text></g><clipPath id="clip-col-labels"><rect x="100" y="300" width="200" height="100"></rect></clipPath><g clip-path="url(#clip-col-labels)"><g transform="translate(200,300)" class="col-label"><text color="black" font-size="10px" transform="rotate(-90)" text-anchor="end" alignment-baseline="middle">a</text></g></g></svg>';
```

Comment:
Apparently with ES6 string templates, you can freely mix newlines into the string and they will show up literally. This line would read a lot easier if we tried that here.